ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -3,34 +3,6 @@ base_images:
     name: "4.13"
     namespace: ocp
     tag: ansible
-  assisted-image-service:
-    name: ocm-2.8
-    namespace: edge-infrastructure
-    tag: assisted-image-service
-  assisted-installer:
```

Comment:
Could you double confirm that only hypershift and no other tests use those images ?

Reply:
yes, i can confirm it, these images all support hypershift-agent workflow